src/ui/settings/colorMode.js

```diff
@@ -0,0 +1,67 @@
+//; -*- mode: rjsx;-*-
+/** @jsx jsx */
+import { jsx, useColorMode } from 'theme-ui';
+import { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { SettingsSection } from './elements';
+
+const ColorModeSettings = () => {
+  const { t } = useTranslation();
+  const [, setColorMode] = useColorMode();
+
+  const getSystemPreference = () => {
+    const isDarkPrefered = window.matchMedia('(prefers-color-scheme: dark)');
+    if(isDarkPrefered.matches) {
+      setColorMode('dark');
+    }
+    else {
```

Comment:
```suggestion
    } else {
```

Studio uses that style.

Reply:
Done.